app/src/main/java/com/google/samples/apps/sunflower/compose/home/HomeScreen.kt

```diff
@@ -169,12 +169,12 @@ private fun HomeTopAppBar(
     CenterAlignedTopAppBar(
         title = {
             Row(
-                Modifier.fillMaxWidth(),
+                Modifier,
```

Comment:
> You can delete this line since the default is Modifier. In fact, you can probably just remove the entire row since an Arrangement.Center doesn't do anything if it's only matching the child's size. Can you try deleting the row and just having the text be the title to verify that looks right?

yes sir you are right it still working same.

Reply:
> You can delete this line since the default is Modifier. In fact, you can probably just remove the entire row since an Arrangement.Center doesn't do anything if it's only matching the child's size. Can you try deleting the row and just having the text be the title to verify that looks right?

Sir with due respect please review my pull request and merge if i have successfully completed the task if not please tell me what else i can do